pkg/activator/throttler.go

```diff
@@ -133,6 +147,20 @@ func (t *Throttler) Try(rev RevisionID, function func()) error {
 	return nil
 }
 
+func (t *Throttler) activatorEndpointsUpdated(newObj interface{}) {
+	endpoints := newObj.(*corev1.Endpoints)
+	count := resources.ReadyAddressCount(endpoints)
+	atomic.StoreInt32(&t.activatorCount, int32(count))
+}
+
+func (t *Throttler) getActivatorCount() int {
```

Comment:
```suggestion
func (t *Throttler) activatorCount() int {
```

nit: that's a bit more idiomatic I think.

Reply:
Updated to use `activatorCount` instead